### ISSUE TITLE
Tooltip/Infotip view truncated

### ISSUE THREAD

**Issue by alexandre-pecorilla** [sparrowwallet/sparrow#1531]
I'm on Debian 12 and I've upgraded to version 2.0.0-1 using the deb package.

The tooltip when hovering stuff isn't working anymore. All I see is a black line. Anyone having the same issue?

**Comment by craigraw** [comment]
I've had no other reports of this. Can you post a screenshot? What window manager are you using?

**Comment by alexandre-pecorilla** [comment]
Thank you for your reply.

I’m using Xfwm4 as my window manager on Debian 12 with XFCE.

There’s not much to show in a screenshot. When I hover over anything, only a thin black line appears, which seems to be the top of the tooltip truncated before any text is visible.

**Comment by alexandre-pecorilla** [comment]
Thank you for your reply.

My window manager is Xfwm4  (Debian 12 XFCE).

Below is a screenshot:

![issue](https://github.com/user-attachments/assets/d3238bbe-f429-41af-9e4b-c38e2d7fad93)

**Comment by craigraw** [comment]
Thanks for the screenshot - it is helpful. I suspect the issue here is that the system font is unable to display at `0.85em`, which is the smaller size used for tooltips. Are you running the default system font?

**Comment by alexandre-pecorilla** [comment]
Yes I havent changed any of my OS/Sparrow's default font settings.

**Comment by time-out** [comment]
Same problem here with same OS Linux deb12 6.1.0-26-amd64, XFCE:
https://github.com/user-attachments/assets/5d709d7a-6d02-40f5-99c8-bc5345233192

**Comment by karlvlam** [comment]
I got the same issue at ubuntu 24.04, XFCE

**Comment by craigraw** [comment]
It appears this has been fixed upstream in https://github.com/openjdk/jfx/pull/1437#pullrequestreview-2058602510, and released in JavaFX 23. The next release of Sparrow will upgrade from JavaFX 22 to JavaFX 23 to fix.

**Comment by AndySchroder** [comment]
I also can't rename accounts on the left sidebar because of this bug.

**Comment by craigraw** [comment]
Upgraded to JavaFX 23.0.1 in 25f441a6. If anyone here can run from source (instructions in the project README), testing is appreciated.

**Comment by craigraw** [comment]
Sparrow 2.1.1 has been released which includes JavaFX 23.0.2. Please test and report back.

**Comment by craigraw** [comment]
Closing as fixed.
